internal/services/cdn/cdn_frontdoor_origin_resource.go

```diff
@@ -389,7 +405,7 @@ func resourceCdnFrontDoorOriginDelete(d *pluginsdk.ResourceData, meta interface{
 	ctx, cancel := timeouts.ForDelete(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	id, err := parse.FrontDoorOriginID(d.Id())
+	id, err := parse.FrontDoorOriginIDInsensitively(d.Id())
```

Comment:
this is coming from the Resource ID and so should be being parsed case-sensitively:

```suggestion
	id, err := parse.FrontDoorOriginID(d.Id())
```

Reply:
Fixed.